client/stylesheets/_toCat.scss

```diff
@@ -46,13 +46,13 @@ GLOBAL | GLOBAL DIVS | HEADER | FOOTER **/
 
 
 .box-title-listing {
+    // border-left: 5px solid $color-spacered;
     background: $color-spacered !important;
-    font-family: inherit;
     color: #FFFFFF;
-    // border-left: 5px solid $color-spacered;
-    padding-left: 4px;
-    margin: 0 !important;
+    font-family: inherit;
     font-size: 1.3rem;
+    margin: 0 !important;
+    padding-left: 4px;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered background, color, font-family, font-size, margin, padding, padding-left